### ISSUE TITLE
AGhost has hands amputated by default

### ISSUE THREAD

**Issue by Aexxie** [space-wizards/space-station-14#21564]
## Description
Using AGhost occasionally brutally cuts off your ghost's hands

**Reproduction**
No clue, seems random. Happens frequent enough that it is an issue. Fixable by restarting the server.

**Screenshots**
![image](https://github.com/space-wizards/space-station-14/assets/125095677/0062baf1-05a9-4772-8ea0-0e3a8b7e9767)

**Additional context**
Found in my local server/dev client. The build should be up to date.

**Comment by Aexxie** [comment]
**Yes** it is actually AGhost, I just removed the sidebar.
![image](https://github.com/space-wizards/space-station-14/assets/125095677/89865312-ac60-4524-884e-d79ce4097b82)

**Comment by Aexxie** [comment]
Update: It seems to happen whenever you reconnect as AGhost. I also found that you can fix by taking control of an entity and AGhosting again

**Comment by deltanedas** [comment]
when you dc i think your ghost is deleted so you get a normal ghost after

**Comment by Aexxie** [comment]
If it does, AGhost persists and cannot be set again

**Comment by ElectroJr** [comment]
> when you dc i think your ghost is deleted so you get a normal ghost after

Yes, it should do this, but you should still be able to aghost after that.

> If it does, AGhost persists and cannot be set again

How are you setting aghost? The command seems to work fine for me.


https://github.com/space-wizards/space-station-14/assets/60421075/1252cc6c-0d3a-4631-b684-3c464d9e36bc

**Comment by Aexxie** [comment]
> > when you dc i think your ghost is deleted so you get a normal ghost after
> 
> Yes, it should do this, but you should still be able to aghost after that.
> 
> > If it does, AGhost persists and cannot be set again
> 
> How are you setting aghost? The command seems to work fine for me.
> 
> 
> https://github.com/space-wizards/space-station-14/assets/60421075/1252cc6c-0d3a-4631-b684-3c464d9e36bc
> 
> 

So sorry about that. I became pretty busy afterwards and didn't see this. I think it is an issue but it isn't nearly as frequent.

Running aghost doesn't fix it when it happens, it just stays as the ghost it was. I don't think it only happens when disconnecting but under other unknown circumstances, too.

It only can be restored after taking control of a mob, then running aghost while you are in control of it.
